### DISCUSSION TITLE
Importar dados de notas de corretagem

### DISCUSSION THREAD

**Discussion by LeoCardosoJr** [Menighin/cei-crawler#43] [Ideas]
Estava verificando, vi que existe um modelo de notas de corretagem, então achei que poderia se enquadrar e ser interessante para o projeto, se conseguisse ler as notas de corretagem e exportar os dados, mais ou menos no mesmo estilo do CEI, para facilitar as declarações de impostos

**Comment by Menighin** [comment]
Oi Leo, boa noite!
Qual tela do CEI vc se refere que tem essa informação? 🤔

**Reply by LeoCardosoJr** [reply]
Boa tarde, ele não faz parte, acho que não se encaixaria nesse projeto né?! A ideia seria ler pdf de notas de corretagem mesmo e transformar em linhas de tabela, que não encontrei nenhum projeto que fizesse isso.

**Reply by Menighin** [reply]
Ah sim...
É, realmente acredito que foge do escopo desse projeto, que é basicamente um crawler do CEI...
Acho que faria mais sentido isso de fato ser uma lib a parte
